src/Microsoft.Extensions.Logging.TraceSource/TraceSourceFactoryExtensions.cs

```diff
@@ -12,39 +13,39 @@ public static class TraceSourceFactoryExtensions
         /// <summary>
         /// Adds a TraceSource logger named 'TraceSource' to the factory.
         /// </summary>
-        /// <param name="factory">The <see cref="LoggerFactory"/> to use.</param>
+        /// <param name="collection">The <see cref="LoggerFactory"/> to use.</param>
         /// <param name="switchName">The name of the <see cref="SourceSwitch"/> to use.</param>
-        public static LoggerFactory AddTraceSource(
-            this LoggerFactory factory,
+        public static ILoggerBuilder AddTraceSource(
+            this ILoggerBuilder collection,
```

Comment:
This argument is called `collection` in some, and `builder` in other similar methods.